app-tracking-protection/vpn-api/src/main/java/com/duckduckgo/mobile/android/vpn/service/VpnStaticNotificationContentPlugin.kt

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2023 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.mobile.android.vpn.service
+
+import android.app.PendingIntent
+import androidx.annotation.LayoutRes
+import androidx.core.app.NotificationCompat
+
+interface VpnStaticNotificationContentPlugin {
```

Comment:
What does it mean `Static` notification?
Should we just call it `VpnReminderNotificationContentPlugin` ?

Reply:
Reminder used to pertain to Disabled notif before so i tried to stay away from the term. But now that I removed anything called "reminder notif", can probably use that.

I just thought of static since unlike the Enabled notif, these notification contents are not designed to change.